### PR TITLE
[DO-NOT-MERGE] Docs for Optimize extension

### PR DESCRIPTION
**Pull request by swarna04**

### CODE REVIEW

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+[identityMap addItem:item withNamespace:@"userCRMID"]; // userCRMID being used as Third Party ID
+
+[AEPMobileEdgeIdentity updateIdentities:identityMap];
+```
+{% endtab %}
+{% endtabs %}
+
+### Analytics for Target (A4T)
+
+Set up the Analytics for Target (A4T) cross-solution integration, by enabling the A4T campaigns, to use Analytics as the reporting source for an activity. Subsequently, all reporting and segmentation for that activity is based on Analytics data collection. For more information, see [Adobe Analytics for Adobe Target (A4T)](https://docs.adobe.com/content/help/en/target/using/integrate/a4t/a4t.html).
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:288]: We don't need the comma there. I'd just go with "Set up the ... integration by enabling the A4T campaigns to use Analytics..."

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+
+AEPExperienceEvent* experienceEvent = [[AEPExperienceEvent alloc] initWithXdm:displayInteractionXdm data:additionalData datasetIdentifier:nil];
+[AEPMobileEdge sendExperienceEvent:event completion:nil];
+```
+{% endtab %}
+{% endtabs %}
+
+## Configuration keys
+
+To update the SDK configuration programmatically, use the following information to change the Optimize extension configuration values. For more information, see [Programmatic updates to Configuration](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/configuration/configuration-api-reference#updateconfiguration).
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:407]: "For more information, see the [programmatic updates to Configuration tutorial].".

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+
+| Key | Required | Description | Data Type |
+| :--- | :--- | :--- | :--- |
+| optimize.datasetId | No | Override dataset's Identifier which can be obtained from the Experience Platform UI. For more details see, [Datasets UI guide](https://experienceleague.adobe.com/docs/experience-platform/catalog/datasets/user-guide.html?lang=en) | String |
+
+{% hint style="info" %}
+If the override dataset is used for proposition tracking, make sure the corresponding schema definition contains the `Experience Event - Proposition Interaction` field group. For more information, see [Setup schemas & datasets](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/62a861aec745af2d8237c287656c68d8f8cdd5ed/getting-started/configure-schema-and-dataset.md).
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:414]: "For more information, see the [setup schemas and datasets] tutorial."

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 4]
```diff
@@ -0,0 +1,542 @@
...
+  // Processing upon registration completion
+}];
+```
+
+{% endtab %}
+{% endtabs %}
+
+## resetIdentities
+
+This `MobileCore` API can also be invoked to clear out the client-side data for Optimize extension, e.g. in-memory propositions cache.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:249]: "... client-side data for the Optimize extension, such as the in-memory propositions cache."

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 5]
```diff
@@ -0,0 +1,542 @@
...
+| :--- | :--- | :--- |
+| class | `DecisionScope` | `AEPDecisionScope` |
+| class | `Proposition` | `AEPProposition` |
+| class | `Offer` | `AEPOffer` |
+
+{% tabs %}
+{% tab title="iOS — Swift" %}
+### DecisionScope
+
+This class represents the decision scope which is used to fetch the decision propositions from the Edge decisioning services. The encapsulated scope name can also represent the Base64 encoded JSON string created using the provided activityId, placementId and itemCount.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:355]: "... represent the Base64-encoded JSON string... provided activityId, placementId, and itemCount.".

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 6]
```diff
@@ -0,0 +1,542 @@
...
+
+    /// Optional Offer characteristics dictionary
+    @objc public let characteristics: [String: String]?
+
+    /// Weak reference to Proposition instance
+    @objc weak var proposition: Proposition?
+}
+```
+
+The `Offer` class extension provides methods for generating XDM data for Proposition Interactions field group which can be used for proposition tracking. It also contains direct methods for  tracking proposition display and tap interactions.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:468]: double space between "for" and "tracking".

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 7]
```diff
@@ -0,0 +1,542 @@
...
+    func displayed() {...}
+
+    /// Dispatches an event for the Edge extension to send an Experience Event to the Edge network with the tap interaction data for the given proposition item.
+    func tapped() {...}
+}
+```
+
+### OfferType
+
+An enum indicating the type of an Offer, derived from the proposition item `format` field in personalization query response.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:499]: I don't think offer needs to be capitalized here. So it can just be "... the type of an offer, derived from..."

**Review by calebk1 — CHANGES_REQUESTED** [review]
Some comments!

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 8]
```diff
@@ -0,0 +1,417 @@
+# Adobe Journey Optimizer - Decisioning Extension
+
+The Adobe Journey Optimizer - Decisioning extension powers real-time personalization workflows using Adobe Journey Optimizer - Offer Decisions or Adobe Target in mobile apps via the Edge Network. It helps deliver personalized decisions to your app and enables tracking user interactions with the proposed decisions.
+
+## Prerequisites
+
+Before starting, make sure the following steps are completed.
+
+* Your IMS organization is provisioned for Edge decisioning. 
+* If using Adobe Target, Target activities are set up in your organization/ workspace on Target UI. For more details, see [Target activities guide](https://experienceleague.adobe.com/docs/target/using/activities/target-activities-guide.html?lang=en).
```
[the remark this answers is not in the document]
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:10]: This wouldn't quite make sense. I'll rephrase it as - ...Target activities are set up in your desired workspace in your organization on Target UI. Hope that explains it better!

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 9]
```diff
@@ -0,0 +1,417 @@
...
+* If using Journey Optimizer - Offer Decisions, Decisions are set up in your organization/ sandbox on Experience Platform UI. For more details, see [Create Decisions](https://experienceleague.adobe.com/docs/offer-decisioning/using/create-manage-activities/create-offer-activities.html?lang=en).
```
[the remark this answers is not in the document]
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:11]: I plan to rephrase this as well - ...Decisions are set up in your desired sandbox in your organization on Experience Platform UI.

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (calebk1) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:10]: Yup, that makes sense to me!

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 10]
```diff
@@ -0,0 +1,417 @@
...
+{% endtab %}
+{% endtabs %}
+
+## Adobe Journey Optimizer - Offer Decisions
+
+{% hint style="warning" %}
+Some offer constraints are currently not supported with the mobile Experience Edge workflows, for example Capping. The Capping field value specifies the number of times an offer can be presented across all users. For more details, see [Offer eligibility rules and constraints guide](https://experienceleague.adobe.com/docs/offer-decisioning/using/managing-offers-in-the-offer-library/creating-personalized-offers.html#eligibility).
+{% endhint %}
+
+### Decision Scope
```
[the remark this answers is not in the document]
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:121]: I'd prefer `Decision Scope`, since I'm trying to explain the concept here. Please refer to this guide - https://experienceleague.adobe.com/docs/experience-platform/edge/personalization/offer-decisioning/offer-decisioning-overview.html?lang=en
I have a separate class reference for the `DecisionScope` class in APi reference

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 11]
```diff
@@ -0,0 +1,417 @@
...
+
+The `DecisionScope` public class provides a convenience initializer to create a scope object using the activityId, placementId and optional itemCount. The Decision scope activity and placement information can be obtained from the Decision on Experience Platform UI.
```
[the remark this answers is not in the document]
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:123]: > A convenience installer? What does that mean?

convenience initializer - It's a Swift terminology. Maybe I can just use the word constructor to keep it generic.

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md (hunk 3 above)
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:414]: Changed to [setup schemas and datasets guide]

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md (hunk 2 above)
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:407]: Changed to [programmatic updates to Configuration guide]

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 12]
```diff
@@ -0,0 +1,417 @@
...
+![Datastream configuration](../../.gitbook/assets/ajo-decisioning-datastream-configuration.png)
+
+### Configure Adobe Journey Optimizer - Decisioning extension in Tag property for Mobile
+
+On [Experience Platform Data Collection](https://launch.adobe.com/), navigate to **Data Collection** > **Tags** using the left navigation panel. Select an existing mobile Tag property or create a new property.
+
+1. In your mobile property, navigate to **Extensions** in the left navigation panel and click on the **Catalog** tab.
+2. In the extensions Catalog, search or locate the **Adobe Journey Optimizer - Decisioning** extension, and click **Install*.
+3. No extension configuration is necessary so click **Save**.
+4. Follow the publishing process to update SDK configuration. For more details, see [Publish the configuration](https://aep-sdks.gitbook.io/docs/getting-started/create-a-mobile-property#publish-the-configuration).
```
[the remark this answers is not in the document]
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:33]: Using [publish the configuration guide]

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 13]
```diff
@@ -0,0 +1,417 @@
...
+
+* Your IMS organization is provisioned for Edge decisioning. 
+* If using Adobe Target, Target activities are set up in your organization/ workspace on Target UI. For more details, see [Target activities guide](https://experienceleague.adobe.com/docs/target/using/activities/target-activities-guide.html?lang=en).
+* If using Journey Optimizer - Offer Decisions, Decisions are set up in your organization/ sandbox on Experience Platform UI. For more details, see [Create Decisions](https://experienceleague.adobe.com/docs/offer-decisioning/using/create-manage-activities/create-offer-activities.html?lang=en).
+
+## Adobe Experience Platform Data Collection setup
+
+### Configure the Datastream for Adobe Target and/ or Journey Optimizer - Offer Decisions
+
+On [Experience Platform Data Collection](https://launch.adobe.com/), navigate to **Data Collection** > **Datatreams** using the left navigation panel. Select an existing datastream or create a new datastream. For more details, see [Configure datastreams](https://aep-sdks.gitbook.io/docs/getting-started/configure-datastreams).
```
[the remark this answers is not in the document]
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:17]: Using [configure datastreams guide]

**Comment by swarna04** [comment]
@calebk1 Thank you for your review and super useful feedback comments. I've incorporated the feedback here. Let me know if you have any further comments.

**Comment by swarna04** [comment]
@calebk1 Checking again! Please let me know if you have any other suggestions on this one, I've incorporated changes from previous feedback.

**Review by calebk1 — APPROVED** [review]
LGTM!

**Comment by swarna04** [comment]
@praschetan Can you please take a look as well and let us know if you have any feedback.

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 14]
```diff
@@ -0,0 +1,952 @@
...
+  // Processing upon registration completion
+}];
+```
+
+{% endtab %}
+{% endtabs %}
+
+## resetIdentities
+
+This `MobileCore` API can also be invoked to clear out the client-side data for the Optimize extension, such as the in-memory propositions cache.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:368]: Relative link, not absolute link.

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 15]
```diff
@@ -0,0 +1,952 @@
...
+    /**
+     * Constructor creates a {@code DecisionScope} using the provided {@code name}.
+     *
+     * @param name {@link String} containing scope name.
+     */
+    public DecisionScope(final String name) {...}
+
+    /**
+     * Constructor creates a {@code DecisionScope} using the provided {@code activityId} and {@code placementId}.
+     * <p>
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:492]: We can remove the `<p>` tags.

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 16]
```diff
@@ -0,0 +1,952 @@
...
+
+    /// Offer content string
+    @objc public let content: String
+
+    /// Optional Offer characteristics dictionary
+    @objc public let characteristics: [String: String]?
+}
+```
+
+The `Offer` class extension provides methods for generating XDM data for Proposition Interactions field group which can be used for proposition tracking. It also contains direct methods for tracking proposition display and tap interactions.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:859]: "... for generating XDM data for the proposition interactions field group, which can be used..."

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 17]
```diff
@@ -0,0 +1,685 @@
...
+{% tabs %}
+{% tab title="Android" %}
+#### Java
+
+```java
+public class Offer {
+  ...
+  /**
+    * Generates a map containing XDM formatted data for {@code Experience Event - Proposition Interactions} field group from this {@code Proposition} item.
+    * <p>
```
[the remark this answers is not in the document]
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:553]: That's needed for the actual Javadoc, but can be removed here 👍

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md (hunk 16 above)
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:859]: Proposition Interactions is the name of the field group, shouldn't we retain the case here?

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 18]
```diff
@@ -0,0 +1,685 @@
...
+
+```java
+implementation 'com.adobe.marketing.mobile:core:1.+'
+implementation 'com.adobe.marketing.mobile:edge:1.+'
+implementation 'com.adobe.marketing.mobile:edgeidentity:1.+'
+implementation 'com.adobe.marketing.mobile:optimize:1.+'
+```
+{% endtab %}
+{% tab title="iOS (AEP 3.x)" %}
+Configure your app target to fetch Mobile Core, Edge, Identity for Edge Network and Optimize from Cocoapods by specifying the following pod dependencies in your `Podfile`.
```
[the remark this answers is not in the document]
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:57]: We don't need to make any updates here. The Podfile will be the same for Swift/ Objective-C apps.

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 19]
```diff
@@ -0,0 +1,685 @@
...
+
+{% hint style="warning" %}
+For AEPOptimize APIs to work properly, it is required that you integrate Mobile Core and Edge extensions as well in your mobile app. For more details see, documentation on [Mobile Core](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core) and [Adobe Experience Platform Edge Network](https://aep-sdks.gitbook.io/docs/foundation-extensions/experience-platform-extension).
+{% endhint %}
+
+### Install the Experience Platform Mobile SDK
+
+{% tabs %}
+{% tab title="Android" %}
+Add the Mobile Core, Edge, Identity for Edge Network and Optimize dependencies in your app's gradle file.
```
[the remark this answers is not in the document]
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:47]: Gradle files use the Groovy DSL so we can skip this change as well.

**Comment by swarna04** [comment]
Thank you for your feedback @calebk1. I've updated the pr with the necessary fixes and also replaced the old assets to reflect datastream v2 changes

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (calebk1) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:561]: I don't think we removed the <p> tags!

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 20]
```diff
@@ -0,0 +1,1005 @@
+# API Reference
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:1]: Lowercase r here is fine.

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 21]
```diff
@@ -0,0 +1,1005 @@
...
+```objc
+[AEPMobileOptimize clearCachedPropositions];
+```
+
+{% endtab %}
+{% endtabs %}
+
+## extensionVersion
+
+The `extensionVersion()` method (on Android) or the `extensionVersion` property (on iOS) return the version information for currently installed AEPOptimize extension.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:57]: "... returns the version information..."

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 22]
```diff
@@ -0,0 +1,952 @@
...
+### Java
+
+#### Syntax
+
+```java
+public static void getPropositions(final List<DecisionScope> decisionScopes, final AdobeCallback<Map<DecisionScope, Proposition>> callback)
+```
+
+* _decisionScopes_ is a List of decision scopes for which propositions are requested.
+* _callback_ `call` method is invoked with propositions map of type `Map<DecisionScope, Proposition>`. If callback is an instance of [AdobeCallbackWithError](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/mobile-core-api-reference#adobecallbackwitherror), and if the operation times out or an error occurs in retrieving propositions, `fail` method is invoked with the appropriate [AdobeError](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/mobile-core-api-reference#adobeerror).
```
[the remark this answers is not in the document]
Comment (calebk1) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:122]: Hi! I don't think this has been fixed either.

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 23]
```diff
@@ -0,0 +1,1005 @@
...
+```objc
+[AEPMobileOptimize onPropositionsUpdate: ^(NSDictionary<AEPDecisionScope*, AEPProposition*>* propositionsDict) {
+  // handle propositions
+}];
+```
+
+{% endtab %}
+{% endtabs %}
+
+## registerExtension(s)
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:313]: A question more out of curiosity than something that's an issue - is there a reason that it's extension on Android and extension**s** on iOS? IE: Why is it not the same between the two platforms?

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 24]
```diff
@@ -0,0 +1,1005 @@
...
+
+```swift
+static func updatePropositions(for decisionScopes: [DecisionScope], 
+                               withXdm xdm: [String: Any]?,
+                               andData data: [String: Any]? = nil)
+```
+
+* _decisionScopes_ is an array of decision scopes for which propositions need updating.
+* _xdm_ additional xdm formatted data to be attached to the Experience Event.
+* _data_ additional freeform data to be attached to the Experience Event.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:431]: These should be sentence-fied, since the previous one is also a sentence.

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md [hunk 25]
```diff
@@ -0,0 +1,1005 @@
...
+
+```objc
++ (void) updatePropositions: (NSArray<AEPDecisionScope*>* _Nonnull) decisionScopes 
+                    withXdm: (NSDictionary<NSString*, id>* _Nullable) xdm
+                    andData: (NSDictionary<NSString*, id>* _Nullable) data;
+```
+
+* _decisionScopes_ is an array of decision scopes for which propositions need updating.
+* _xdm_ additional xdm formatted data to be attached to the Experience Event.
+* _data_ additional freeform data to be attached to the Experience Event.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:458]: Same comment here - this should be a sentence.

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md (hunk 16 above)
Comment (calebk1) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:912]: Oh it is? Then yes, it should retain the case then.

**Review by calebk1 — COMMENTED** [review]
Just a few more comments!

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 26]
```diff
@@ -0,0 +1,693 @@
+# Adobe Journey Optimizer - Decisioning Extension
+
+The Adobe Journey Optimizer - Decisioning extension powers real-time personalization workflows using Adobe Journey Optimizer - Offer Decisioning or Adobe Target in mobile apps via the Edge Network. It helps deliver personalized decisions to your app and enables tracking user interactions with the proposed decisions.
+
+## Prerequisites
+
+Before starting, make sure the following steps are completed.
+
+* Your IMS organization is provisioned for edge decisioning.
```
[the remark this answers is not in the document]
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:9]: Noted, ty!

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md [hunk 27]
```diff
@@ -0,0 +1,693 @@
...
+![Datastream configuration - Offer Decisioning](../../.gitbook/assets/ajo-decisioning-datastream-configuration-od.png)
+
+3. For Adobe Target, navigate to **Adobe Target** section and enable it. Specify the configuration. Make sure to configure the required information like Client Code.
+
+![Datastream configuration - Adobe Target](../../.gitbook/assets/ajo-decisioning-datastream-configuration-at.png)
+
+4. Click **Save**.
+
+
+
```
[the remark this answers is not in the document]
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/README.md:31]: Nope, let's remove those.

File: using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md (hunk 23 above)
Comment (swarna04) [reply using-mobile-extensions/adobe-journey-optimizer-decisioning/api-reference.md:313]: That's because it was introduced during Swift rewrite, we might introduce it on Android too with the next major update.

**Comment by swarna04** [comment]
@calebk1 I've updated the pr based on your latest feedback comments. Thank you.

**Review by calebk1 — APPROVED** [review]
LGTM!
